modules/govuk/manifests/apps/content_performance_manager.pp

```diff
@@ -122,6 +122,14 @@
     port => $redis_port,
   }
 
+  if $::domain =~ /^.*\.(dev|integration\.publishing\.service|staging\.publishing\.service)\.gov\.uk/ {
```

Comment:
Isn't the more normal thing to do here to add it to the enviornment-specific config files? eg https://github.com/alphagov/govuk-puppet/blob/master/hieradata/development.yaml#L97-L183